akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -251,7 +252,7 @@ private[akka] object StreamLayout {
          * materialized value computation of its submodules.
          */
         Atomic(this),
-        Attributes.none)
+        DefaultAttributes.nested)
```

Comment:
If `Attributes.none` is used, then actors get `unknown-operation` in their names. I think it is better to be exclusive about it and let actor name reflect how the module was constructed, that materialized to the particular actor.